fedimintd/src/fedimintd.rs

```diff
@@ -121,6 +120,9 @@ struct ServerOpts {
     #[arg(long, env = FM_API_URL_ENV)]
     api_url: Option<SafeUrl>,
 
+    #[arg(long, env = FM_ENABLE_IROH_ENV)]
+    enable_iroh: bool,
```

Comment:
I wonder if clap accepts `--enable-iroh=false` if we eventually make it the default. Worst case we can just switch to `--disable-iroh` I guess.